nevergrad/benchmark/experiments.py

```diff
@@ -1233,6 +1234,39 @@ def neuro_control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experime
                     yield xp
 
 
+@registry.register
+def olympus_surfaces(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Olympus surfaces """
+    funcs = []
+    for kind in [
+        "AckleyPath",
+        "Dejong",
+        "HyperEllipsoid",
+        "Levy",
+        "Michalewicz",
+        "Rastrigin",
+        "Rosenbrock",
+        "Schwefel",
+        "StyblinskiTang",
+        "Zakharov",
+    ]:
+        for k in range(2, 6):
```

Comment:
Maybe 5 is enough because dim = 1 000 000 in a noisy context will be a lot.

Reply:
okay I will fix that. what about the noise scale ?